lightning/src/routing/test_utils.rs

```diff
@@ -131,7 +167,7 @@ pub(super) fn id_to_feature_flags(id: u8) -> Vec<u8> {
 	}
 }
 
-pub(super) fn build_line_graph() -> (
+pub(super) fn build_line_graph(node_features: NodeFeatures) -> (
```

Comment:
nit: Rename to make it consistent:
```suggestion
pub(super) fn build_line_graph_with_features(node_features: NodeFeatures) -> (
```